earth_enterprise/src/server/wsgi/wms/ogc/xml/v111/capabilities.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/env python2.7
```

Comment:
Just taking one of these as a point, and question.  I see you had decided to go with env.